app/views/overview/summary1.html

```diff
@@ -10,7 +10,7 @@
     <h1 class="govuk-heading-l">
     Confirm conversion overview information</h1>
     <p>This information is pre-populated from TRAMS and the school's application form.</p>
-    <p>TRAMS last updated on: 23 March 2021</p>
+    <p class="govuk-body-s"></p>TRAMS last updated on: 23 March 2021</p>
```

Comment:
I don't think this code is correct, it will need to be:
`<p class="govuk-body-s">TRAMS last updated on: 23 March 2021</p>`

Reply:
Cool thanks!